inference-engine/tests/functional/plugin/cpu/single_layer_tests/concat.cpp

```diff
@@ -201,10 +201,9 @@ INSTANTIATE_TEST_SUITE_P(smoke_Concat3D, ConcatLayerCPUTest,
 
 INSTANTIATE_TEST_SUITE_P(smoke_Concat_1D_2D, ConcatLayerCPUTest,
                         ::testing::Combine(
-                                ::testing::Values(0),
-                                ::testing::Values(std::vector<std::vector<size_t>>{{2, 4},
-                                                                                   {3, 4}},
-                                                  std::vector<std::vector<size_t>>{{2}, {3}}),
+                                ::testing::Values(1),
+                                ::testing::Values(std::vector<std::vector<size_t>>{{3, 2},
+                                                                                   {3, 10}}),
```

Comment:
looks like you have deleted 1d case, I think because now it works inPlace. Therefore I we should add dedicated 1d tests anyway since we have to check such case, even though it is inPlace now.

Reply:
Yes, great! [Added](https://github.com/openvinotoolkit/openvino/blob/0f068a3589dc0b58d84e8efc61d2a1c3e8d1b776/inference-engine/tests/functional/plugin/cpu/single_layer_tests/concat.cpp#L226). Thanks!